README.md

```diff
@@ -277,18 +277,25 @@ Compose.prototype.map = function(f) {
 };
 ```
 
-#### `sequence` method
+#### `traverse` method
 
 ```hs
-sequence :: Apply f, Traversable t => t (f a) ~> (b -> f b) -> f (t a)
+traverse :: Apply f, Traversable t => t a ~> ((a -> f b), (c -> f c)) -> f (t b)
```

Comment:
I asked the question [here][1] too, since the type signature was added in #147.


[1]: https://github.com/fantasyland/fantasy-land/pull/147/files#r79309177